examples/transforms/transform.rs

```diff
@@ -0,0 +1,113 @@
+use bevy::prelude::*;
+
+use std::f32::consts::PI;
+
+struct CubeState {
+    start_pos: Vec3,
+    move_speed: f32,
+    turn_speed: f32,
+}
+
+struct Center {
+    min_size: f32,
+}
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(move_cube.system())
+        .add_system(rotate_cube.system())
+        .add_system(scale_sphere_proportional_to_cube_travel_distance.system())
+        .run();
+}
+
+fn move_cube(mut cubes: Query<(&mut Transform, &mut CubeState)>, timer: Res<Time>) {
+    for (mut transform, cube) in cubes.iter_mut() {
+        let forward = transform.forward();
+        transform.translation += forward * cube.move_speed * timer.delta_seconds();
+    }
+}
+
+fn rotate_cube(mut cubes: Query<(&mut Transform, &mut CubeState)>, timer: Res<Time>) {
+    for (mut transform, cube) in cubes.iter_mut() {
+        let full_turn = transform
+            .looking_at(Vec3::ZERO, transform.local_y())
+            .rotation;
+        let incremental_turn_weight = cube.turn_speed * timer.delta_seconds();
+        transform.rotation = transform.rotation.lerp(full_turn, incremental_turn_weight);
+    }
+}
+
+fn scale_sphere_proportional_to_cube_travel_distance(
+    mut transformabels: QuerySet<(
+        Query<(&Transform, &CubeState)>,
+        Query<(&mut Transform, &Center)>,
+    )>,
```

Comment:
You dont need to use a QuerySet, if you can prove that the queries are disjoint.
```suggestion
    cube: Query<(&Transform, &CubeState), Without<Center>>,
    center: Query<(&mut Transform, &Center)>,
    )>,
```

Reply:
That's much clearer!

Reply:
I'm curious, is there a specific performance gain either way?

Reply:
I would be astonished if it's measurable.

Reply:
This helped a lot, thanks!